guides/micronaut-aws-secretsmanager/micronaut-aws-secretsmanager.adoc

```diff
@@ -0,0 +1,107 @@
+= @guideTitle@
+
+@guideIntro@
+
+Authors: @authors@
+
+Micronaut Version: @micronaut@
+
+include::{commondir}/common-gettingStarted.adoc[]
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+include::{commondir}/common-create-app.adoc[]
+
+== OAuth 2.0 Dependency
+
+To use OAuth 2.0 integration, add the next dependency:
+
+dependency:micronaut-security-oauth2[groupId=io.micronaut.security]
+
+== Micronaut AWS Secrets manager dependency
+
+Add the following dependencies:
+
+:dependencies:
+
+dependency:micronaut-aws-secretsmanager[groupId=io.micronaut.aws,version=@micronaut-aws-secretsmanagerVersion@]
+dependency:micronaut-aws-sdk-v2[groupId=io.micronaut.aws,version=@micronaut-aws-sdk-v2Version@]
+
+:dependencies:
+
+== Bootstrap.yml
+
+To enable distributed configuration, create a `src/main/resources/bootstrap.yml` configuration file:
+
+resource:bootstrap.yml[]
+
+* You need to enable `micronaut.config-client.enabled`
+* Moreover, we need to set the application name in `bootstrap.yml` instead of `application.yml` so that its value is available for Bootstrap Context compatible beans.
```

Comment:
Don't think it is necessary to mention Bootstrap Context compatible beans which may just add more confusion. Rewrite to:

```
In addition, we need to set the application name in `bootstrap.yml` instead of `application.yml` so that it is available when reading configuration from distributed sources.
```